active/0000-daemonize.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: 2015-05-26
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+ember-cli will run as a daemon in the background, rather than being started for each `ember` command.
+
+# Motivation
+
+Currently running an `ember` command can take quite some time, especially when a lot of addons are installed.
+This is mainly due to the "boot up" period, when Node.js / io.js loads ember-cli itself and *all* the addons.
+Node.js / io.js unnecessarily compiles / loads a huge amount of files that very likely aren't relevant to the task.
+
+# Detailed design
+
+Instead of rebooting ember-cli everytime an `ember` command is executed, there would be a service daemon running in the background
+that performs the tasks. The major advantage is that his daemon would only have to load *once* and could run all subsequent
+tasks in an instant.
+
+The `ember` binary is merely interface to the daemon. It's sole task is proxying the entered task to the daemon,
+which then in turn does all the heavy lifting.
+
+# Drawbacks
+
+##### Yet another background service
+
+However, this isn't a backgroudn service that auto starts on system boot, but only when it is invoked by the user.
+
+##### Bigger risk of memory leaks
+
+Memory leaks would add up over time.
+
+##### Different UX
+
+Users would have to learn how to and get accustomed to start and stop the daemon, depending on how ths is implemented.
```

Comment:
I think they only need to get used to `stopping` it, as it should auto-start on issue of a command that requires it like `build` or `start`